site/src/theme/components/utils/mui-css-baseline-theme-options.ts

```diff
@@ -151,9 +151,7 @@ export const MuiCssBaselineThemeOptions: Components["MuiCssBaseline"] = {
           }
 
           body {
-            overflow: auto;${
-              "" /** @todo: find a pernament solution for preventing the navbar from shifting to the right when a modal is opened on an overflowing page */
-            }
+            overflow-x: hidden !important;
```

Comment:
Why is this still necessary?

Reply:
Since header is `100vw` and gets a padding same as scrollbar, is overflows on x axis as much as scrollbar width if scrollbar is visible. But this prevents the jumping on header.